docs/compilation.rst

```diff
@@ -185,46 +187,99 @@ Prerequisites
     ./configure && make
 
 
-Get the DALI source
-+++++++++++++++++++
+Build DALI
+^^^^^^^^^^
+
+1. Get DALI source code:
 
 .. code-block:: bash
 
-  git clone --recursive https://github.com/NVIDIA/dali
-  cd dali
+  git clone --recursive https://github.com/NVIDIA/DALI
+  cd DALI
 
-Make the build directory
-++++++++++++++++++++++++
+2. Create a directory for CMake-generated Makefiles. This will be the directory, that DALI's built in.
 
 .. code-block:: bash
 
   mkdir build
   cd build
 
+3. Run CMake. For additional options you can pass to CMake, refer to :ref:`OptionalCmakeParamsAnchor`.
 
-Compile DALI
-^^^^^^^^^^^^
+.. code-block:: bash
 
-Building DALI without LMDB support:
-+++++++++++++++++++++++++++++++++++
+  cmake -D CMAKE_BUILD_TYPE=Release ..
+
+4. Build. You can pass ``-j`` option to execute it in several threads
 
 .. code-block:: bash
 
-  cmake ..
   make -j"$(nproc)"
 
+.. _PythonBindingsAnchor:
+
+Install Python bindings
++++++++++++++++++++++++
 
-Building DALI with LMDB support:
-++++++++++++++++++++++++++++++++
+In order to run DALI using Python API, you need to install Python bindings
 
 .. code-block:: bash
 
-  cmake -DBUILD_LMDB=ON ..
-  make -j"$(nproc)"
+    cd build
+    pip install dali/python
+
+.. note::
 
+  Although you can create a wheel here by calling ``pip wheel dali/python``, we don't really recommend doing so. Such whl is not self-contained (doesn't have all the dependencies) and it will work only on the system where you built DALI bare-metal. To build a wheel that contains the dependencies and might be therefore used on other systems, follow :ref:`DockerBuilderAnchor`.
 
-Building DALI using Clang (experimental):
-+++++++++++++++++++++++++++++++++++++++++
+Verify the build (optional)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Obtain test data
+++++++++++++++++
+
+You can verify the build by running GTest and Nose tests. To do so, you'll need DALI_extra repository, which contains test data
+
+.. code-block:: bash
+
```

Comment:
Add hint about LFS installation.

Reply:
Added reference to DALI_extra's doc. I guess it makes do